frontend/src/util/label.test.js

```diff
@@ -0,0 +1,52 @@
+import { render } from '@testing-library/react';
+import { getPlayerLabel, LABEL_NUMERIC, LABEL_ALPHABETIC, LABEL_ROMAN, LABEL_CUSTOM, renderLabel } from "./label";
+
+describe('getPlayerLabel', () => {
+
+    it('returns numeric label correctly', () => {
+        expect(getPlayerLabel(0, LABEL_NUMERIC)).toBe(1);
```

Comment:
Working on moving the `getPlayerLabel` logic to the backend at the moment.

Reply:
I've deprecated the function to prevent ourselves from using it in the future. Also, do you have the specific issue (if exists) that describes moving that logic to the backend?